lte/cloud/go/services/subscriberdb/servicers/subscriberdb_servicer.go

```diff
@@ -76,6 +85,10 @@ func (s *subscriberdbServicer) SyncSubscribers(
 	ctx context.Context,
 	req *lte_protos.SyncSubscribersRequest,
 ) (*lte_protos.SyncSubscribersResponse, error) {
```

Comment:
This servicer method is getting just a bit long. Can you move the "get sub changeset" logic to its own method?